Support level bugfix (#881)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where **generate-docs** was inserting unnecessary escape characters.
* Fixed an issue in the **update-release-notes** command where changes to the pack_metadata were not detected.
* Fixed an issue where **validate** did not check for missing release notes in old format files.

> * Fixed an issue where **validate** command asked for tests for non xsoar supported packs.

Reviewer: ```suggestion
* Fixed an issue where **validate** command failed when no tests were given for a partner supported pack.
```
